modules/apps/object/object-dynamic-data-mapping-storage/src/main/java/com/liferay/object/dynamic/data/mapping/storage/internal/ObjectDDMStorageAdapter.java

```diff
@@ -215,12 +212,16 @@ private long _getObjectDefinitionId(Long ddmStructureId)
 
 			DDMFormField ddmFormField = ddmFormValue.getDDMFormField();
 
+			if (!objectFieldNames.contains(ddmFormField.getFieldReference())) {
+				continue;
+			}
```

Comment:
@gabrielwas, does this code intend to ignore form fields without mapping any mapping to the object associated to the form?

Reply:
Yes. For now, the fieldReference must exist in the selected object (as an object field) in order to store the data.

Reply:
This will change as soon as map the fields through forms UI